oasislmf/cli/model.py

```diff
@@ -343,7 +348,8 @@ def action(self, args):
             accounts_profile_fp=accounts_profile_fp,
             fm_aggregation_profile_fp=aggregation_profile_fp,
             ri_info_fp=ri_info_fp,
-            ri_scope_fp=ri_scope_fp
+            ri_scope_fp=ri_scope_fp,
+            user_data_dir=user_data_dir
```

Comment:
You could follow PEP8 and add a trailing comma so that future additions of parameters only create a single line diff.

Reply:
We use Flake8 to lint the code, and I didn't see any PEP8 warnings/errors about the lack of a trailing comma.

Reply:
From https://www.python.org/dev/peps/pep-0008/#when-to-use-trailing-commas:
> When trailing commas are redundant, they are often helpful when a version control system is used, when a list of values, arguments or imported items is expected to be extended over time. The pattern is to put each value (etc.) on a line by itself, always adding a trailing comma, and add the close parenthesis/bracket/brace on the next line. 

Apparently Flake8 doesn't report it but you can add the `flake8-commas` plugin which will report "C813 - missing trailing comma in Python 3" in cases like this (if I've understood correctly). It's a minor detail.

Reply:
You're free to add trailing commas in the appropriate places in your contributions, but we can't guarantee that this will be standard practice, because different people in the team have different approaches to writing the code - for example, I always use trailing commas when writing multi-line import statements within parentheses, but not in multi-line method calls, but other members write and indent imports and method calls differently etc. We try to be consistent but 100% consistency isn't always possible.

Also, as indicated in the PEP8 guide, the comment on trailing commas is really a suggestion in special cases, not a general recommendation, which is why it does not appear in the canonical list of PEP8 or Flake8 errors or warnings.